src/history/hs.c

```diff
@@ -484,6 +473,9 @@ __wt_hs_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MUL
     btree_id = btree->id;
     __wt_modify_vector_init(session, &modifies);
 
+    /* Disable bulk loads into history store. */
+    __wt_cursor_disable_bulk(session, ((WT_CURSOR_BTREE *)cursor)->btree);
```

Comment:
The better approach here would be to call this wrapped in `WT_WITH_BTREE`, similar to https://github.com/wiredtiger/wiredtiger/blame/develop/src/txn/txn.c#L678

Rather than needing to add a btree to the exclusive_on and exclusive_off APIs. I wonder if it's safe to remove the btree parameter from `__wt_cursor_disable_bulk` too?

Reply:
And use S2BT instead of passing it?

Reply:
Done